spec/features/users_can_create_projects_spec.rb

```diff
@@ -0,0 +1,13 @@
+require "rails_helper"
+
+RSpec.feature "Users can create a new project" do
+
```

Comment:
I think this space will cause a linter fail - is the linter part of your standard testing?